zubhub_frontend/zubhub/src/views/PageWrapper.jsx

```diff
@@ -89,6 +89,11 @@ function PageWrapper(props) {
   const classes = useStyles();
   const common_classes = useCommonStyles();
   const trigger = useScrollTrigger();
+
+  //Created new state "show" to show and hide the scroll to top button
+  const [show, setShow] = useState(false);
```

Comment:
Remove comment and change the name to `showScrollBar`, `setShowScrollBar`. We should aim to made code more descriptive and comments should remain for only complex decisions int he code that have been taken or to describe APIs well.

Reply:
Ok I will do it.